BetterMD/elements/title.py

```
@@ -0,0 +1,27 @@
+from typing import Text
```

Ruff (0.8.2)

No such file or directory (os error 2)

(E902)